EG-4699: Added some missing steps, examples, updated version numbers listed. (#395)
corda/corda-docs: content/en/docs/corda-os/4.4/building-the-docs.md

Steps:

1. Download [Visual Studio Code](https://code.visualstudio.com/) or a markdown editor of your choice ([atom](https://atom.io/), for example).

> 2. Download [Hugo](https://github.com/gohugoio/hugo/releases). Use the latest, extended version, otherwise at least v0.65.

Reviewer: Perhaps use the format in this readme (the intro para but whatever else you think would be helpful):

https://github.com/corda/engineering-kb#install-hugo

3. Ensure the Hugo binary is on your `PATH`.
4. Fork the [corda-docs](https://github.com/corda/corda-docs/) repository and add it as upstream.

> 5. Clone your fork locally and, in command-line, navigate (`cd`) to root of the fork repo.

Reviewer: ```suggestion
5. Clone your fork locally.
6. Open a a console / command prompt window and navigate (`cd`) to the root directory of the fork repo.
```

6. Run `hugo serve`.

> 7. Open the local docs site build on [http://localhost:1313](http://localhost:1313) (or whatever it says in the console) in your browser.

Reviewer: ```suggestion
7. Open the local docs site built on [http://localhost:1313](http://localhost:1313) in your browser.
```

> upstream	git://github.com/corda/corda-docs.git (push)

Reviewer: I think this may be an error (also in the older version of that instruction where I worked from back in the day when updating this page of 4.4+) - the upstream entries should also be in https:// format. Also, my name should not be anywhere in the public docs apparently... another mistake:)) 
Can you replace with:

`origin	https://github.com/my-github-username/corda-docs.git (fetch)  #  YOUR FORK`
`origin	https://github.com/my-github-username/corda-docs.git (push)`
`upstream	https://github.com/corda/corda-docs.git (fetch)  #  THE ORIGINAL REPO`
`upstream	https://github.com/corda/corda-docs.git (push)`

```

> Alternatively, if you are accessing GitHub without an `ssh` key:

Reviewer: This is again for https and not ssh access? One of the two should become ssh...

Author: Good spot! Fixed in upcoming commit.

```
$ git remote -v

origin	https://github.com/my-github-username/corda-docs.git (fetch)   # YOUR FORK
origin	https://github.com/my-github-username/corda-docs.git (push)
upstream	https://github.com/corda/corda-docs.git (fetch)      # THE ORIGINAL REPO
upstream	https://github.com/corda/corda-docs.git (push)
```

### Remove the upstream repo

> To remove the `upstream` repo:

Reviewer: ```suggestion
If you need to remove the `upstream` repo for any reason:
```

Reviewer: NB This should be amended in our instructions in the KB as well as this not a normal use case and it may be misleading if we leave if this way

```
$ git remote rm upstream
```

### Keep the upstream repo updated

To keep the upstream updated (in other words, to `fetch` all the stuff from the upstream repo):

> `$ git fetch upstream`

Reviewer: Please can you remove the $ prefixes from all commands, not sure why they are there in the first place...
